src/commonMain/kotlin/fr/acinq/lightning/NodeParams.kt

```diff
@@ -189,6 +189,7 @@ data class NodeParams(
             Feature.Wumbo to FeatureSupport.Optional,
             Feature.StaticRemoteKey to FeatureSupport.Mandatory,
             Feature.AnchorOutputs to FeatureSupport.Optional, // can't set Mandatory because peers prefers AnchorOutputsZeroFeeHtlcTx
+            Feature.RouteBlinding to FeatureSupport.Optional,
```

Comment:
Should we force this to be activated in `init` with a `require`, like other features that must be activated? This would be more consistent since we implicitly use route blinding without checking the feature bit.

Reply:
Done